src/Service/DynamoDb/tests/Unit/DynamoDbClientTest.php

```diff
@@ -104,7 +104,7 @@ public function testDeleteItem(): void
 
         $input = new DeleteItemInput([
             'TableName' => 'Foobar',
-
+            'Key' => ['ID' => ['S' => 'foobar']],
```

Comment:
Why is this needed?

Reply:
hehe... previous code had a "bug"

`Key` is required.. But because previously we initialized maps with an empty array (see https://github.com/async-aws/aws/pull/873/files#diff-324c911f8afbcd579bb76e189d1af7df134c2f519be2414f97d830934ec23588L119) it never triggered the test `if ($this->key === null) {throw Exception(...);}`

Reply:
Okey. Can you also update the changelogs with this?